lib/mollie/method.rb

```diff
@@ -17,24 +17,15 @@ class Method < Base
 
     attr_accessor :id,
                   :description,
-                  :amount,
```

Comment:
It's possible to filter methods by `currency` and `value`. Can you add support for this to the Ruby client as well? And add an example of this to the `website-docs` folder?